src/libraries/System.Configuration.ConfigurationManager/src/System.Configuration.ConfigurationManager.csproj

```diff
@@ -275,6 +276,7 @@
     <Reference Include="System.Runtime.Serialization.Formatters" />
     <Reference Include="System.Security.Cryptography.Algorithms" />
     <Reference Include="System.Security.Cryptography.Primitives" />
+    <Reference Include="System.Security.Cryptography.Xml" />
```

Comment:
To my knowledge, S.S.C.Xml is an OOB from Microsoft.NETCore.App, so this seems like it will be adding an unmet dependency.  It should be a package reference, I think?